app/assets/javascripts/navbar.js

```diff
@@ -0,0 +1,167 @@
+                  <Link to="/users">Users</Link>
+                </Menu.Item>
+                <Menu.Item key="/teams">
+                  {" "}
+                  <Link to="/teams">Teams</Link>
+                </Menu.Item>
+                <Menu.Item key="/projects">
+                  {" "}
+                  <Link to="/projects">Projects</Link>
+                </Menu.Item>
+                <Menu.Item key="/tasks">
+                  {" "}
+                  <Link to="/tasks">Tasks</Link>
+                </Menu.Item>
+                <Menu.Item key="/taskTypes">
+                  {" "}
+                  <Link to="/taskTypes">Task Types</Link>
+                </Menu.Item>
+                <Menu.Item key="/scripts">
+                  {" "}
+                  <Link to="/scripts">Scripts</Link>
+                </Menu.Item>
+              </SubMenu>,
+              <SubMenu
+                key="sub2"
+                title={
+                  <span>
+                    <Icon type="line-chart" />Statistics
+                  </span>
+                }
+              >
+                <Menu.Item key="/tasks/overview">
+                  <Link to="/tasks/overview">Overview</Link>
+                </Menu.Item>
+                <Menu.Item key="/statistics">
+                  <Link to="/statistics">Weekly</Link>
+                </Menu.Item>
+              </SubMenu>,
+              <SubMenu
+                key="sub3"
+                title={
+                  <span>
+                    <Icon type="medicine-box" />Help
+                  </span>
+                }
+              >
+                <Menu.Item key="/help/faq">
+                  <a target="_blank" href="/help/faq">
+                    FAQ
+                  </a>
+                </Menu.Item>
+                <Menu.Item key="11">
+                  <a target="_blank" href="/assets/docs/frontend-api/index.html">
+                    Frontend API Documentation
+                  </a>
+                </Menu.Item>
+                <Menu.Item key="/help/keyboardshortcuts">
+                  <a target="_blank" href="/help/keyboardshortcuts">
+                    keyboard Shortcuts
+                  </a>
+                </Menu.Item>
+              </SubMenu>,
+              <Menu.Item key="13">
+                <a href="https://discuss.webknossos.org" target="_blank" rel="noopener noreferrer">
+                  <Icon type="notification" />
+                  Discussion Board
+                </a>
+              </Menu.Item>,
+              <SubMenu
+                key="sub4"
+                className="pull-right"
+                title={
+                  <span>
+                    <Icon type="user" />
+                    {`${this.props.activeUser.firstName} ${this.props.activeUser.lastName}`}
+                  </span>
+                }
+              >
+                <Menu.Item key="resetpassword">
+                  <Link to="/reset">Reset Password</Link>
```

Comment:
To me it looks like that the user is logged in at this point, right? If a user is already logged in, he has the option to change his password with `/api/changepw`. `/reset` is just in case someone forgot his password (and thus not logged in).

Reply:
Yes and no :-)
At this point the user is indeed logged in. The link `/reset` will route the UI to the reset view and nothing more. Please remember, only routes with a `/api/...` prefix will invoke server actions. 
In this case, however, you are right the link to navigate to the "change password view".